src/Views/Admin/Pages/templates/setup-template.php

```diff
@@ -0,0 +1,100 @@
+<div class="wrap" class="give-setup-page">
+
+	<h1 class="wp-heading-inline">
+		<?php echo __( 'Setup GiveWP', 'give' ); ?>
+	</h1>
+
+	<hr class="wp-header-end">
+
+	<!--
+	<div class="error">
+		&nbsp;
+	</div>
+	-->
+
+	<section class="section">
+		<h2 class="section-header">Create Your First Donation Form in Minutes</h2>
+
+		<!-- Onboarding Wizard -->
+		<?php
+		$this->render_template(
+			'setup-step-template',
+			[
+				'dashicon'   => 'admin-settings',
+				'labelText'  => 'First-Time Configuration',
+				'actionText' => 'Get Started',
+			]
+		);
+		?>
+
+	</section>
+
+	<section class="section">
+		<h2 class="section-header">Connect a Payment Gateway to Accept Donations</h2>
+
+		<!-- PayPal Checkout -->
+		<?php
+		$this->render_template(
+			'setup-step-template',
+			[
+				'dashicon'   => 'admin-generic',
+				'labelText'  => 'PayPal Checkout',
+				'actionText' => 'Connect to PayPal',
+			]
+		);
+		?>
+
+		<!-- Stripe -->
+		<?php
+		$this->render_template(
+			'setup-step-template',
+			[
+				'dashicon'   => 'admin-generic',
+				'labelText'  => 'Stripe',
+				'actionText' => 'Connect to Stripe',
+			]
+		);
+		?>
+
+	</section>
+
+	<section class="section">
+		<h2 class="section-header">Level Up Your Fundraising</h2>
+
+		<!-- Give 101 -->
+		<?php
+		$this->render_template(
+			'setup-step-template',
+			[
+				'dashicon'       => 'welcome-learn-more',
+				'labelText'      => 'GiveWP 101',
+				'actionText'     => 'View GiveWP 101',
+				'actionLocation' => 'https://givewp.com/documentation/',
```

Comment:
@mathetos @DevinWalker Do we have to add short link here?

Reply:
This is a placeholder URL, which I can update with a tracked link.